### DISCUSSION TITLE
Elwynn Forest Battered Chest grouping Wrath

### DISCUSSION THREAD

**Discussion by ghost** [TrinityCore/TrinityCore#30294] [General]
Just wondering if anyone has issues with this grouping for pooling? If so please give suggestions. Also comment if you are fine with it.

![image](https://github.com/user-attachments/assets/ec6795fb-a845-4652-b333-7fca43744311)

**Comment by ghost** [comment]
The two outside mine spawns in C and E could possibly be in a group? That would give all 4 spawn groups except for B "Forest Edge"

**Comment by ghost** [comment]
Like this:
![image](https://github.com/user-attachments/assets/3cd7dcf4-33d6-479f-9086-0a90d198a629)

**Comment by shamage** [comment]
For which versions of the retail edition are you finding this?
Because, when I search my memory, they were available until the release of BFA or until Shadowlads - here I vaguely remember that I still met them. The current release 11.0.2.56819 no longer has them. However, it has something else and that is :  
![Ashampoo_Snap_streda 2  októbra 2024_11h55m30s](https://github.com/user-attachments/assets/fbddf2f1-16dd-4af6-b43a-b42abb37f8c7)

this is the only one I came across from the whole area. I was horrified by this reduction of objects.

**Comment by ghost** [comment]
I'm spawning 335.

**Comment by ghost** [comment]
I have been leveling a Mage in Elwynn on Cata and haven't seen a chest yet.

**Reply by shamage** [reply]
> I have been leveling a Mage in Elwynn on Cata and haven't seen a chest yet.

It is possible that this type of object / chest is from release version DF or later. That's why I asked for which version you are looking for spawns. 

Edit :  In this case, do not look for this chest, it is quite possible that it will not be available for older versions like DF or Shadowland.

**Reply by ghost** [reply]
I'm sorry. With all my Chest issues for 335 I just assumed people would know. I added wrath to the title.
